src/components/formatters/include/formatters/CFormatterJsonBase.h

```diff
@@ -99,7 +99,8 @@ class CFormatterJsonBase {
    */
   static void jsonValueToObj(
```

Comment:
@jacobkeeler you added new parameter. I think that description should be updated